assets/sass/local.sass

```diff
@@ -387,4 +387,13 @@ dt.summary-list__key__wider
   th
     border-bottom: none
   td
-    border-bottom: none
\ No newline at end of file
+    border-bottom: none
+
+.email
+  padding-top: 8px
+
+.user-heading
+  padding-left: 59px
```

Comment:
This is a bit rubbish -  because we're not using a standard component and have some kind of table/radio button crossover. Need to push the name "column header" past the length of a radio button